go.mod

```diff
@@ -285,3 +285,6 @@ replace github.com/containerd/containerd => github.com/containerd/containerd v1.
 
 // https://issues.redhat.com/browse/OCPBUGS-8540
 replace go.mongodb.org/mongo-driver => go.mongodb.org/mongo-driver v1.11.2
+
+// https://issues.redhat.com/browse/OCPBUGS-25508
+replace vbom.ml/util => github.com/fvbommel/util v0.0.2
```

Comment:
why do we need this replace? Not sure what this is used for. I see vbom.ml in our go.sum but `go mod why` does not turn it up

Reply:
we needed it as `go get golang.org/x/crypto@v0.17.0` command unrecognized `vbom.ml/util`. This is being imported in `github.com/openshift/library-go`. yeah, it didn't show as it was not existed. 
Note that installer module is not imported this `vbom.ml/util`.

Reply:
> we needed it as `go get golang.org/x/crypto@v0.17.0` command unrecognized `vbom.ml/util`. This is being imported in `github.com/openshift/library-go`. yeah, it didn't show as it was not existed. Note that installer module is not imported this `vbom.ml/util`.

Hm I can't reproduce the issue locally. It may be due to my local cache containing `vbom.ml/util`. Will check.